cli/src/main.rs

```diff
@@ -580,7 +587,7 @@ fn run() -> Result<(), CliError> {
                                 .required(true)
                                 .takes_value(true)
                                 .value_name("name")
-                                .help("The name of the template"),
+                                .help("Template name"),
```

Comment:
This isn't consistent with the help strings above (those are "Name of X")

Reply:
Addressed